docs/changes/latest.inc

```diff
@@ -30,5 +30,7 @@ Bugs
 
 - Fix target transformers that only normal transformers are wrapped (:gh:`94` by `Sami Hamdan`_)
 
+- Fix classification bug due to scikit-learn version 0.2.4 (:gh:`#108` by `Sami Hamdan`_)
```

Comment:
sklearn version is 0.24. Was it a bug? Or something that changed within sklearn?

I would say something like "Fix compatibility with new scikit-learn release 0.24"

Reply:
You are right this is a better way of putting it. 
We just needed an additional attribute `classes_` for classification